.doc_gen/metadata/s3_metadata.yaml

```diff
@@ -3073,3 +3073,21 @@ s3_Scenario_ObjectLock:
                 - S3LockWorkflow.dotnetv3.S3ActionsWrapper
   services:
     s3: {PutObjectLockConfiguration, PutObjectRetention, GetObjectRetention, PutObjectLegalHold, GetObjectLegalHold, GetObjectLockConfiguration}
+s3_SelectObjectContent:
+  title: Use SQL with &S3; Select to retrieve a subset of data by using an &AWS; SDK
+  title_abbrev: Use SQL with &S3; Select
+  synopsis: retrieve a subset of data using SQL.
+  category:
+  languages:
+    Java:
+      versions:
+        - sdk_version: 2
+          github: java/example_code/s3
+          sdkguide:
+          excerpts:
+            - description: Use the <ulink url="https://sdk.amazonaws.com/java/api/latest/software/amazon/awssdk/services/s3/S3AsyncClient.html#selectObjectContent(software.amazon.awssdk.services.s3.model.SelectObjectContentRequest,software.amazon.awssdk.services.s3.model.SelectObjectContentResponseHandler)"><code>selectObjectContent</code></ulink> method of the <code>S3AsyncClient</code> to select a subset of a JSON object. The following example shows a query using a JSON object. The <ulink url="https://github.com/awsdocs/aws-doc-sdk-examples/blob/main/javav2/example_code/s3/src/main/java/com/example/s3/async/SelectObjectContentExample.java">complete example</ulink> also shows the use of a CSV object.

```

Comment:
This is up to you, but this description feels mostly redundant to me. The selectObjectContent API link is the same as the auto-generated one at the end of the example and the link to GitHub is (nearly) the same as the one in the "more on GitHub" note. Do you really need these extra links and text?

Reply:
I do want to keep the last link, because it takes the user directly to the code being shown rather then them having to divine where it is the big mass of Java examples.